packages/appstore/src/components/app.tsx

```diff
@@ -22,19 +23,21 @@ const App = ({ passthrough: { WS, root_store } }: TAppProps) => {
 
     return (
         <CashierStoreProvider store={root_store as any}>
-            <StoreProvider store={root_store as any}>
-                <main
-                    className={classNames('dashboard', {
-                        'theme--light': !ui.is_dark_mode_on,
-                        'theme--dark': ui.is_dark_mode_on,
-                        'dashboard-onboarding': window.location.pathname === routes.onboarding,
-                    })}
-                >
-                    <div className='dw-dashboard'>
-                        <Routes />
-                    </div>
-                </main>
-            </StoreProvider>
+            <CFDStoreProvider store={root_store as any}>
```

Comment:
Can't we use TRootStore here ?

Reply:
`TRootStore` is deprecated

<img width="818" alt="image" src="https://user-images.githubusercontent.com/129206554/234501438-e0705a25-75fa-4f44-b242-b9f2f5890985.png">

cc: @farzin-deriv